src/slots/BooleanInSlot.js

```diff
@@ -0,0 +1,37 @@
+const Joi = require('joi')
+
+const JOIous = require('../mixins/instanceMixins/JOIous')
+const InSlot = require('./InSlot')
+
+class BooleanInSlot extends InSlot {
+
+  constructor({ defaultValue = null, ...slotData }) {
+    super(slotData)
+    this.dataType = BooleanInSlot.DATA_TYPE
+    this.defaultValue = defaultValue
+  }
```

Comment:
the `constructor` is an instance method,
which is why, I assume, you tend to see it defined below the `static` methods with the other instance methods. Certainly not a blocking request. I did not open this comment to write that though, so I digress.

Should `BooleanInSlot`s (and others) ever only be `tareable: true`? If so, do you think it would be appropriate to update the `SCHEMA` to reflect that?

Reply:
changed the order; removed tareable from boolean and oneOf slots.